### PR TITLE
Add gimbal measurement and estimation data packet to dataflash logs

### PR DESCRIPTION
**Pull request by bugobliterator**

### CODE REVIEW

**Comment by arthurbenemann** [comment]
This needs to be tested, and merged. Please let's do this before Gimbal PVT (end of Jun)

File: libraries/AP_Mount/AP_Mount_MAVLink.cpp [hunk 1]
```diff
@@ -104,6 +148,11 @@ void AP_Mount_MAVLink::handle_gimbal_report(mavlink_channel_t chan, mavlink_mess
 {
     _gimbal.update_target(_angle_ef_target_rad);
     _gimbal.receive_feedback(chan,msg);
+    Log_Write_Gimbal(_gimbal);
```
Comment (arthurbenemann) [inline libraries/AP_Mount/AP_Mount_MAVLink.cpp]: Is this saving the GIMBAL_REPORT messages at 100Hz?

File: libraries/AP_Mount/AP_Mount_MAVLink.cpp [hunk 2]
```diff
@@ -104,6 +148,11 @@ void AP_Mount_MAVLink::handle_gimbal_report(mavlink_channel_t chan, mavlink_mess
...
+    if(!_params_saved) {
+        _frontend._externalParameters.fetch_params();       //last parameter save might not be stored in dataflash so retry
```
Comment (arthurbenemann) [inline libraries/AP_Mount/AP_Mount_MAVLink.cpp]: Can we save the parameters received from the gimbal on the datalog as well?

**Comment by arthurbenemann** [comment]
@bugobliterator Comments in the code, please review.

**Comment by jschall** [comment]
Merged to solo-U1.1 branch.

**Pull request closed without merging**